opennms-base-assembly/src/main/filtered/etc/opennms.properties

```
@@ -869,3 +869,4 @@ org.opennms.newts.nan_on_counter_wrap=true
 
 # By default hostnames are determined for a node's IP addresses during the provisioning's audit phase.
 # org.opennms.provisiond.reverseResolveRequisitionIpInterfaceHostnames=true
+flow.templates.dir=${flow.templates.dir}
```

Please prefix the property:
```suggestion
flow.templates.dir=${org.opennms.flow.templates.dir}
```